[*] running method core_negotiate_tlv_encryption
[*] Negotiating TLV encryption
[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100a771cbcbe71e464a1ea931621157273c99b86f02a022084d1a8905121328692e94358ee5952624abc0999e25ea9c466c51b3c630b64e8286a9228bea558187e57125e1e70f0572caa209ae0cb39e1c3b067d8ea443cc0302cc3629c6d99f2303c5d2a832994c04ae0b210401120945c32db36522d8e1fb195d5a4dfa8306dfff9badfeb31ee3917a0ff3cd0c6b74680ac687ca609368f9f2291bd011ce404eacc7d3d47742b2aa4ec5bac6ca7e65b3f1f44840775e3b054dfbce0249873f0d9b0e63e1b7a0490c08bd28db5eb0aefa259db9f35d03d9744ba4d14feb64f9c89ebb3b92492b365afa5847624c252476e8c3a111b6fba8e4098017886b331a8d730203010001
[*] AES key: 0x65a9d55d8aa5a0b3acf118d6a16c23bb0330ac9a18e715f38008984e5987e227
[*] Encrypted AES key: 0x28354d09e67b0a59f20d9d04abebb697c031ec7d7358064346ab673b0e7fe93f45e41bc18f49219f86d5d014f34482a441ddb2ca10afad2a51390d05a25e9be4b09e350eb2c22b66adbad637c486ce06e31925edcad47cf14b03b8ba6fdbb5594b5873be8ef6301b1db1977fef535b673527aa6de825cbdaa4d95f90e74487e339a3e6afbfd0efc5b5030a90b3681ae3c5faf49d926e408c4bc4ea934fc192399dfa77fdec3a82394a08f207c014e1204a11bc63f8e53729a364acb1bbf37b2807a366b00ab495af5073504549132aacb15e12cbd11b02d0d4628d637684fcc210511dccea77a8fe716c8e37d8d33eca8293a0706f69f8c78d5c64a69fe0bb1
[*] TLV encryption sorted
[*] sending response packet
[*] running method core_set_session_guid
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_loadlib
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getuid
[*] sending response packet
[*] running method stdapi_sys_config_sysinfo
[*] sending response packet
[*] running method core_set_uuid
[*] sending response packet
[*] running method stdapi_net_config_get_interfaces
[*] sending response packet
[*] running method stdapi_net_config_get_routes
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_separator
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-pxghzobl'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 1 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 2 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 2
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 3 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 3
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 4 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 4
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 5 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 5
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 6 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 6
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 7 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 7
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method stdapi_fs_file_move
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 8 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 8
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 9 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 9
[*] sending response packet
[*] running method stdapi_fs_ls
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo DGHJKBFL']
[*] added process id: 3061, handle: 1
[*] added channel id: 10 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 11 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 11
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
[*] added process id: 3066, handle: 2
[*] added channel id: 12 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 13 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 13
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
[*] added process id: 3072, handle: 3
[*] added channel id: 14 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 3
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 15 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 15
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 16 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 16
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 17 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 17
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 18 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 18
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 19 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 19
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
